tests/TestApplication/config/sylius_adyen.yaml

```diff
@@ -0,0 +1,34 @@
+sylius_adyen:
+    payment_methods:
+        allowed_types:
```

Comment:
Shouldn't we define them for the entire plugin, not just for the test application?